trackbar_base.h

```diff
@@ -8,17 +8,16 @@ using TrackbarString = std::basic_string<TCHAR>;
  * Class for host of trackbar control to (optionally) implement to receive callbacks.
  * \see TrackbarBase::set_callback
  */
-class NOVTABLE TrackbarCallback {
-public:
-    /**
-     * Called when thumb position changes.
-     *
-     * \param [in]    pos        Contains new position of track bar thumb
-     * \param [in]    b_tracking Specifies that the user is dragging the thumb if true
-     *
-     * \see TrackbarBase::set_range
-     */
-    virtual void on_position_change(unsigned pos, bool b_tracking) {};
+class NOVTABLE TrackbarCallback{public :
```

Comment:
The new formatting here isn't right.

Reply:
Resolved – `NOVTABLE` needed to be added to `AttributeMacros` in `.clang-format`.